project-base/app/src/DataFixtures/Demo/DemoDataFactory/ProductDemoDataFactory.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\DataFixtures\Demo\DemoDataFactory;
+
+use App\DataFixtures\Demo\UnitDataFixture;
+use App\DataFixtures\Demo\VatDataFixture;
+use App\Model\Product\ProductData;
+use App\Model\Product\ProductDataFactory;
+use App\Model\Product\Unit\Unit;
+use DateTime;
+use Shopsys\FrameworkBundle\Component\DataFixture\PersistentReferenceFacade;
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Shopsys\FrameworkBundle\Model\Pricing\Vat\Vat;
+
+class ProductDemoDataFactory
```

Comment:
I think I would rather see this class in the framework package but let's discuss this in the team.

Reply:
not implementing this as we discussed